### PR TITLE
feat: add the meilisearch webapp service

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**
Update search.html to read for meilisearch.

### CODE REVIEW

File: .github/workflows/ci_cd.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ env:
   PACKAGE_NAME: 'ansys-sphinx-theme'
   PACKAGE_NAMESPACE: 'ansys_sphinx_theme'
   DOCUMENTATION_CNAME: 'sphinxdocs.ansys.com'
+  MEILISEARCH_API_KEY: ${{ secrets.MEILISEARCH_API_KEY}}
```
Comment (Revathyvenugopal162) [inline .github/workflows/ci_cd.yml:16]:
```suggestion
  MEILISEARCH_API_KEY: ${{ secrets.MEILISEARCH_API_KEY }}
```

File: src/ansys_sphinx_theme/theme/ansys_sphinx_theme/components/search-field.html [hunk 2]
```diff
@@ -8,47 +9,45 @@
...
-        inputSelector: '#search-bar-input, #indexUidSelector',
-        enableDarkMode: 'auto',
-        debug: true // Set debug to true if you want to inspect the dropdown
+      var theSearchBar = docsSearchBar({
+          hostUrl: "{{ HOST_URL }}",
+          apiKey: "{{ API_KEY }}",
+          indexUid: "{{ theme_use_meilisearch.index_uids.items()|first|first }}",
+          inputSelector: '#search-bar-input',
+          enableDarkMode: 'auto',
+          debug: true // Set debug to true if you want to inspect the dropdown
```
Comment (MaxJPRey) [inline src/ansys_sphinx_theme/theme/ansys_sphinx_theme/components/search-field.html:37]: @Revathyvenugopal162 I am assuming we will set it to `false` later on.

Comment (Revathyvenugopal162) [reply src/ansys_sphinx_theme/theme/ansys_sphinx_theme/components/search-field.html:37]: yes, may be after the inspection .

**Review by akaszynski — APPROVED** [review]
Thanks for adding this. Let's get this out and merged and start expanding the usage.

Some followups
- [ ] Fully transition to new backend (move search.pyansys.com to use our new container service). If this is done already, great!
- [ ] Shutdown search VM [meilisearch-host](https://portal.azure.com/#@ansys.com/resource/subscriptions/47f35875-b811-4339-87f1-f3364983a690/resourceGroups/PYANSYS/providers/Microsoft.Compute/virtualMachines/meilisearch-host/overview)
- [ ] Longer term: create tool to generate index json so we can stop using the page scraper.

**Pull request merged**